src/core/structures/acroform/PDFAcroText.ts

```diff
@@ -1,10 +1,46 @@
-import { PDFDict } from 'src/core';
+import { PDFDict, PDFName, PDFNumber, PDFObject } from 'src/core';
 import { PDFTerminalField } from './index';
 
 class PDFAcroText extends PDFTerminalField {
   static fromDict(dict: PDFDict): PDFAcroText {
     return new PDFAcroText(dict);
   }
+
+  RV(): PDFObject | undefined {
+    return this.dict.lookup(PDFName.RV);
+  }
+
+  MaxLen(): PDFNumber | undefined {
+    return this.dict.lookupMaybe(PDFName.MaxLen, PDFNumber);
+  }
+
+  isMultiLine(): boolean {
+    return !!(this.Ff().value() & (1 << 13));
```

Comment:
Should these be changed to use `this.getFlags()`?

Reply:
Updated